src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.cs

```diff
@@ -673,7 +673,7 @@ private static CultureData CreateCultureWithInvariantData()
                 return CultureData.Invariant;
             }
 
-            if (GlobalizationMode.PredefinedCulturesOnly && !GlobalizationMode.Invariant)
+            if (GlobalizationMode.PredefinedCulturesOnly)
             {
```

Comment:
could you please add assert here for !GlobalizationMode.Invariant?

Reply:
Done.